drivers/retained_mem/Kconfig.nrf

```diff
@@ -8,3 +8,10 @@ config RETAINED_MEM_NRF_GPREGRET
 	help
 	  Enable driver for Nordic nRF GPREGRET-based retained memory
 	  register support.
+
+config RETAINED_MEM_NRF_RAM_CTRL
+	bool "nRF RAM retention driver"
+	default y
+	depends on DT_HAS_ZEPHYR_RETAINED_RAM_ENABLED
+	help
```

Comment:
this needs `depends on SOC_NRF54L15` or `depends on SOC_SERIES_NRF54LX` (whichever if applicable)

Reply:
Also needs to depend on `RETAINED_MEM_ZEPHYR_RAM`

Reply:
done